App.tsx

```diff
@@ -1,3 +1,5 @@
+import GlobalStore from './state_manage/store';
+import { Provider } from 'react-redux'; 
 import React from 'react';
```

Comment:
There's another import from 'react' in line 8, you can combine them. You can don't worry about alphabetizing these inputs unless you want to, I'll be setting up ES Lint that do it automatically.